docs/Using Price Feeds/get-the-latest-price.html

```diff
@@ -87,6 +85,24 @@
     </div>
 </div>
 
+## Solana network examples
+
+Chainlink Price Feeds run live on the [Solana Devnet](https://explorer.solana.com/?cluster=testnet) with sub-second updates. Chainlink’s Solana deployment has no dependencies on external blockchain networks such as Ethereum.
+
+The program address for this price data feed is `2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE` on the Solana Devnet. You can view the program in the [Solana Devnet Explorer](https://explorer.solana.com/address/2yqG9bzKHD59MxD9q7ExLvnDhNycB3wkvKXFQSpBoiaE?cluster=devnet).
```

Comment:
Where did you get this program ID from?

Reply:
I think ideally we will say something like:
```
This demo will show you how to:
1. Deploy your own program that uses Chainlink Data Feeds
2. Interact with an account that stores state from a deployed program

Please note, that for a solidity comparison, the combination of an account + program is equivalent to a smart contract on an EVM chain. The demo will show you how to work with a program you deploy, but you can refactor the client section to work with a program Id of your choice, as state and logic are separated in Solana. 
```

Reply:
@mccordnate recommended that we highlight that program ID. It's also the owner ID for all the Devnet feeds: https://docs.chain.link/docs/solana-price-feeds/

For example, here is the BTC / USD feed:
https://explorer.solana.com/address/6dbkV6QCToTk6DRfuJyrGuz18kZ4rPUSHLLLVrryWdUC?cluster=devnet

Reply:
Yeah, given the way Solana works, the demo is more important for how to actually use the price feeds than the actual program ID. I think it's just useful context for users to check against, though.

Thinking on this again in the morning, it's probably fine to leave it with the direct explorer link to the program.

Reply:
I clarified these points and included the primer on what to expect from the demo.